symbolic-minidump/src/cfi.rs

```diff
@@ -284,7 +286,14 @@ impl<W: Write> AsciiCfiWriter<W> {
     fn process_breakpad(&mut self, object: &BreakpadObject<'_>) -> Result<(), CfiError> {
         for record in object.stack_records() {
             match record? {
-                BreakpadStackRecord::Cfi(r) => writeln!(self.inner, "STACK CFI {}", r.text),
+                BreakpadStackRecord::Cfi(r) => match r {
```

Comment:
I'm wondering if the printing logic in this function should be moved to a `Display` impl.